docs/book/src/topics/addons.md

```diff
@@ -279,6 +279,22 @@ The following resources need to be updated when using `capi-quickstart.yaml` (th
                 .
     ```
 
+### Disable Azure network discovery (if using custom images)
+
+By default, Azure's images are coming with a `cloud-init` configuration that enables automatic network discovery mechanism that interferes with the Azure CNI.
+
+In fact, it assigns secondary IP Configurations to the host os, which we do not want. The Azure CNI needs those free to allocate them to pod netns's/veth's)
```

Comment:
It assigns secondary IP Configurations to the host OS, which causes problems because the Azure CNI needs those free to allocate them to pod netns's/veth's.

Reply:
Done!